package.json

```diff
@@ -36,8 +36,7 @@
   },
   "homepage": "https://github.com/saucelabs/node-testcomposer#readme",
   "dependencies": {
-    "axios": "^1.7.5",
-    "form-data": "^4.0.0"
+    "axios": "^1.7.5"
```

Comment:
Could you also update the engine slightly? [According to the docs](https://nodejs.org/api/globals.html#class-formdata), that's when the native version was added for the first time.
```
  "engines": {
    "node": ">=16.15.0"
  },
```

Reply:
👍